### PR TITLE
TargetNet

### PR DESCRIPTION
**Pull request by katarinagresova**

### CODE REVIEW

File: src/miRBench/encoder.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ class TargetNetEncoder():
     U = [0, 0, 0, 1, 0]
     T = [0, 0, 0, 1, 0]
     - = [0, 0, 0, 0, 1]
-    First, gene sequence is truncated to 40nt nad miRNA sequence is reversed to be in 5' to 3' direction.
+    First, gene sequence is truncated to 40nt and reversed.
```
Comment (stephaniesamm) [inline src/miRBench/encoder.py]: I would add "to be in the 3' to 5' direction", in agreement with https://academic.oup.com/bioinformatics/article/38/3/671/6408435#394284742:~:text=%2C-,where,in%20the%205%E2%80%B2%2Dto%2D3%E2%80%B2%20and%203%E2%80%B2%2Dto%2D5%E2%80%B2%20directions%2C%20respectively.,-We%20use%20subscript

File: src/miRBench/encoder.py [hunk 2]
```diff
@@ -234,8 +234,8 @@ def prepare_dataset(self, df, miRNA_col, gene_col, batch_size=64, shuffle=False,
 
         for _, row in df.iterrows():
 
-            mirna_seq = reverse(row[miRNA_col].upper().replace("T", "U"))
-            mrna_seq = row[gene_col].upper().replace("T", "U")[0:40]
+            mirna_seq = row[miRNA_col].upper().replace("T", "U")
+            mrna_seq = reverse(row[gene_col].upper().replace("T", "U")[0:40])
```
Comment (stephaniesamm) [inline src/miRBench/encoder.py:238]: Are you defining the reverse function somewhere else Kat?

Comment (katarinagresova) [reply src/miRBench/encoder.py:238]: Yes, it is here: https://github.com/katarinagresova/miRBench/blob/d68442ccbd82ebfe7bc297b989af5dda025d49bd/src/miRBench/encoder.py#L376

File: src/miRBench/encoder.py (hunk 1 above)
Comment (katarinagresova) [reply src/miRBench/encoder.py]: Added.

**Pull request merged**